submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/model/ext/SectionExt.kt

```diff
@@ -4,3 +4,6 @@ import ac.uk.ebi.biostd.persistence.common.NO_TABLE_INDEX
 import ac.uk.ebi.biostd.persistence.model.Section
 
 internal fun Section.isTableElement() = tableIndex != NO_TABLE_INDEX
+
+internal val Section.title: String?
+    get() = attributes.find { it.name == "Title" }?.value
```

Comment:
I think we have a constant for this one, havent we?

Reply:
Done